### PR TITLE
Remove explore redux

### PR DESCRIPTION
**Pull request by AdityaHegde**

### CODE REVIEW

File: src/lib/svelte-query/globalQueryClient.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+/**
+ * This is temporary until everything is moved to using svelte-query
```
Comment (ericpgreen2) [inline src/lib/svelte-query/globalQueryClient.ts:2]: Why is this temporary?

File: src/lib/application-state-stores/explorer-stores.ts [hunk 2]
```diff
@@ -1,7 +1,41 @@
...
+  leaderboardMeasureId: string;
+  leaderboards?: Array<LeaderboardValues>;
+  filters: MetricViewRequestFilter;
+  selectedCount?: number;
+  // user selected time range
+  selectedTimeRange?: TimeSeriesTimeRange;
+  // this marks whether anything related to this explore is stale
+  // this is set to true when any measure or dimension changes.
+  // this also is set to true when related model and its dependant source updates (TODO)
+  isStale?: boolean;
```
Comment (ericpgreen2) [inline src/lib/application-state-stores/explorer-stores.ts:37]: We can remove `isStale` now, right?

File: src/lib/svelte-query/queries/metric-view.ts [hunk 3]
```diff
@@ -181,6 +182,15 @@ export const getMetricViewTotalsQueryKey = (
   return [TotalsId, metricsDefId, isReferenceValue];
 };
 
+export const invalidateMetricView = async (
+  queryClient: QueryClient,
+  metricsDefId: string
+) => {
+  // wait for meta to be invalidated
+  await queryClient.invalidateQueries([MetaId, metricsDefId]);
```
Comment (ericpgreen2) [inline src/lib/svelte-query/queries/metric-view.ts:190]: Does this `await` wait for the refetch to complete? If so, then we're good. If not, then I wonder if we'll have a race condition when `invalidateMetricViewData` runs.

**Review by ericpgreen2 — APPROVED** [review]
Left a couple notes, otherwise looks good 👍

File: src/lib/svelte-query/globalQueryClient.ts (hunk 1 above)
Comment (AdityaHegde) [reply src/lib/svelte-query/globalQueryClient.ts:2]: I am hoping we can pass this from components when calling various methods.

File: src/lib/application-state-stores/explorer-stores.ts (hunk 2 above)
Comment (AdityaHegde) [reply src/lib/application-state-stores/explorer-stores.ts:37]: Yes

File: src/lib/svelte-query/queries/metric-view.ts (hunk 3 above)
Comment (AdityaHegde) [reply src/lib/svelte-query/queries/metric-view.ts:190]: This seems to be the case after checking with breakpoints.

**Pull request merged**